sessions/recap-project-5/art-gallery-app/README.md

```diff
@@ -49,228 +43,222 @@ Download a [Favorite Icon](https://lucide.dev/icons/) and save it in an "assets"
 
 #### Acceptance Criteria
 
-- [ ] A navigation link labeled "Favorites" is displayed
-- [ ] Clicking the "Favorites" shows the FavoritesPage
-- [ ] All favorite art pieces are displayed as a list
-- [ ] Each art piece's image is displayed
-- [ ] Each art piece's title is displayed
-- [ ] Each art piece's artist is displayed
-- [ ] Each art piece's is displayed with active favorite-button
+- [ ] A navigation link labeled "Favorites" is displayed.
+- [ ] Clicking "Favorites" shows the Favorites page.
+- [ ] All **favorite** art pieces are displayed in a list format.
+- [ ] Each art piece's image is displayed.
+- [ ] Each art piece's title is displayed.
+- [ ] Each art piece's artist name is displayed.
+- [ ] Each art piece has an active favorite button.
 
 #### Tasks
 
-- [ ] Create the page `pages/favorites` that renders the `ArtPieces` component
-- [ ] Use data from the `artPiecesInfo` state to filter for all favorite art pieces
-- [ ] Pass the list of all favorite art pieces via prop `pieces` to the `ArtPieces` component
+- [ ] Reuse an existing component to display only the favorite art pieces.
+- [ ] Write tests to verify that the navigation link labeled "Favorites" is displayed in all pages.
 
----
+### 6. Comments for Art Pieces
 
-### User Story 7: Comments for Art Pieces
+To enhance user engagement, we’ll add a commenting feature that allows users to write and view comments on each art piece. A comment section will be displayed on the details page, showing comments specifically for the art piece being viewed.
 
 #### Value Proposition
 
 **As an** art enthusiast
 
-**I want to** write comments for art pieces
+**I want to** write comments on art pieces
 
-**so that** I can note my ideas regarding the work.
+**so that** I can note down my thoughts and ideas about the work.
 
 #### Acceptance Criteria
 
-- [ ] The detail view has a list of comments for this art piece with the headline "Comments"
-- [ ] Each comment's text is displayed
-- [ ] Each comment's date and time is displayed
-- [ ] The detail view has an input field to write a comment
-- [ ] The detail view has a submit button labeled "Send"
-- [ ] After submitting the form, the comment is appended to the list of comments
+- [ ] The detail view includes a section labeled "Comments" displaying a list of comments for this art piece.
+- [ ] Each comment’s text is displayed.
+- [ ] Each comment’s date and time are displayed.
+- [ ] The detail view includes an input field for writing a comment.
+- [ ] The detail view includes a submit button labeled "Send".
+- [ ] After submitting the form, the new comment is added to the list of comments.
 
 #### Tasks
 
-- [ ] Store comments per art piece in the `artPiecesInfo` state
-- [ ] Create a `CommentForm` component
-- [ ] `CommentForm` props: `onSubmitComment`
-- [ ] Create a `Comments` components
-- [ ] `Comments` props: `comments`
+- [ ] Create at least two new components for managing and displaying comments.
```

Comment:
Maybe we can be a bit more explicit here as well. Especially on how to store the comments in a global state.

Reply:
Added it as a hint.